extensions/wikia/HTTPSSupport/HTTPSSupportHooks.class.php

```diff
@@ -16,13 +16,10 @@ class HTTPSSupportHooks {
 	const VIGNETTE_IMAGES_HTTP_UPGRADABLE = '#(images|img|static|vignette)(\d+)?\.wikia\.(nocookie\.)?(net|com)#i';
 
 	public static function onMercuryWikiVariables( array &$wikiVariables ): bool {
```

Comment:
Now this method is so simple I think we can just try to use wfHttpsAllowedForURL/wfHttpToHttps there: https://github.com/Wikia/app/blob/b3d119e46384a373d2510de23eff22c001d67cca/extensions/wikia/MercuryApi/models/MercuryApi.class.php#L115 and remove this hook listener